messaginginapp/src/test/java/io/customer/messaginginapp/testutils/extension/GistExtensions.kt

```diff
@@ -16,17 +14,6 @@ fun pageRuleContains(route: String): String = "^(.*$route.*)\$"
 
 fun pageRuleEquals(route: String): String = "^($route)\$"
 
-fun decodeOptionsString(options: String): Message {
```

Comment:
This isn't used anywhere so instead of fixing `instanceId` here, I simply removed it